corehq/apps/registry/static/registry/js/registry_list.js

```diff
@@ -0,0 +1,109 @@
+hqDefine("registry/js/registry_list", [
+    'jquery',
+    'underscore',
+    'knockout',
+    'hqwebapp/js/initial_page_data',
+], function (
+    $,
+    _,
+    ko,
+    initialPageData
+) {
+    let getBadgeText = function (count, singular, plural) {
+        return interpolate(ngettext(singular, plural, count), {"count": count}, true);
+    }
+    let getAcceptedText = function(registry) {
+        return getBadgeText(
+            registry.accepted_invitation_count,
+            "%(count)s Invitation Accepted",
+            "%(count)s Invitations Accepted",
+        )
+    }
+    let getPendingText = function(registry) {
+        return getBadgeText(
+            registry.pending_invitation_count,
+            "%(count)s Invitation Pending",
+            "%(count)s Invitations Pending",
+        )
+    }
+    let getRejectedText = function(registry) {
+        return getBadgeText(
+            registry.rejected_invitation_count,
+            "%(count)s Invitation Rejected",
+            "%(count)s Invitations Rejected",
+        )
+    }
+    let getParticipatorCountText = function(registry) {
+        return getBadgeText(
+            registry.participator_count,
+            "%(count)s Project Space Participating",
+            "%(count)s Project Spaces Participating",
+        )
+    }
+    let DeleteModal = function (element) {
```

Comment:
It's interesting to pass in the element - usually knockout doesn't directly manipulate the DOM in js, other than in custom bindings. The more typical way to do this in our code is to use the `openModal` binding defined in `knockout_bindings.ko.js` and used in places like `releases_table.html`. That said, this code is easy to follow and seems fine.

Reply:
I'll have a look at that